### PR TITLE
Bug 2000837: Update RBAC api to v1 from v1beta1 (backport to 4.8)

### PR DESCRIPTION
**Pull request by priyanka19-98**
Signed off by: Priyanka Jiandani <pjiandan@redhat.com>

### CODE REVIEW

**Comment by gnufied** [comment]
@priyanka19-98 you can't use https://bugzilla.redhat.com/show_bug.cgi?id=1995653 for 4.8. You will have to clone the bug to 4.8 version and then use that bug for this PR.

File: deploy/rbac.yaml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ---
 kind: Role
-apiVersion: rbac.authorization.k8s.io/v1beta1
+apiVersion: rbac.authorization.k8s.io/v1
 metadata:
   name: local-storage-operator
 rules:
```
Comment (gnufied) [inline deploy/rbac.yaml:6]: While we are doing this - should we also update the CRDs?

Comment (priyanka19-98) [reply deploy/rbac.yaml:6]: CRDs are already updated to v1.

**Comment by hardys** [comment]
[patch-manager] Approving for cherry-pick with a score of 0.20 - bz prio medium, I can see the corresponding change was made to master via #270

**Pull request merged**